XLS-0081d-permissioned-dex/README.md

```diff
@@ -0,0 +1,592 @@
+
+This spec proposes an additional flag, to indicate whether the offer should consider only the `DomainID`'s Permissioned DEX or also include the open DEX.
+
+### 4.2. Failure Conditions
+
+The existing set of failure conditions for `OfferCreate` will continue to exist.
+
+The following will be added, if the `DomainID` field is included:
+* The domain doesn't exist.
+* The offer is not a valid domain offer.
+
+### 4.3. State Changes
+
+The existing set of state changes for `OfferCreate` will continue to exist.
+
+If the `DomainID` is included in the `OfferCreate` transaction, the DEX will use the domain-specific orderbook to process the offer.
+
+## 5. Transaction: `Payment`
+
+A `Payment` transaction represents a transfer of value from one account to another, and can involve currency conversions and crossing the orderbook. This transaction type already exists on the XRPL, but is being extended as a part of this spec to also support permissioned DEX domains.
+
+### 5.1. Fields
+
+<details>
+<summary>
+
+As a reference, [here](https://xrpl.org/docs/references/protocol/transactions/types/payment) are the existing fields for the `Payment` transaction.
+</summary>
+
+| Field Name | Required? | JSON Type | Internal Type | Description |
+|------------|-----------|-----------|---------------|-------------|
+|`Amount`|✔️|`string`|`Amount`|The maximum amount of currency to deliver. For non-XRP amounts, the nested field names MUST be lower-case. If the `tfPartialPayment` flag is set, deliver _up to_ this amount instead.|
+|`DeliverMin`| |`string`|`Amount`|The minimum amount of destination currency this transaction should deliver. Only valid if this is a partial payment. For non-XRP amounts, the nested field names are lower-case.|
+|`Destination`|✔️|`string`|`AccountID`|The unique address of the account receiving the payment.|
+|`DestinationTag`| |`number`|`UInt32`|An arbitrary tag that identifies the reason for the payment to the destination, or a hosted recipient to pay.|
+|`InvoiceID`| |`string`|`Hash256`|An arbitrary 256-bit hash representing a specific reason or identifier for this payment.|
+|`Paths`| |`array`|`PathSet`|Array of payment paths to be used for this transaction. Must be omitted for XRP-to-XRP transactions.|
+|`SendMax` | |`string` or `object`|`Amount`|Highest amount of source currency this transaction is allowed to cost, including transfer fees, exchange rates, and [slippage](http://en.wikipedia.org/wiki/Slippage_%28finance%29). Does not include the XRP destroyed as a cost for submitting the transaction. For non-XRP amounts, the nested field names MUST be lower-case. Must be supplied for cross-currency/cross-issue payments. Must be omitted for XRP-to-XRP payments.|
+</details>
+
+We propose these additions:
+
+| Field Name | Required? | JSON Type | Internal Type | Description |
+|------------|-----------|-----------|---------------|-------------|
+|`DomainID`| |`string`|`Hash256`|The domain that the offer must be a part of.|
```

Comment:
Do you mean not offer but account? Which account must be part of the domain? Src? Dst? Both?